src/sentry/auth/access.py

```diff
@@ -80,11 +84,17 @@ def scopes(self) -> FrozenSet[str]:
     def permissions(self) -> FrozenSet[str]:
         pass
 
+    # TODO(cathy): remove this
```

Comment:
Can we create a Jira ticket for this and link in the PR description?

Reply:
i may or may not need to remove it because this is used to return a singular org role which is being utilized by the frontend, but i'll make ticket